build-tools/packages/build-tools/src/fluidBuild/tasks/leaf/flubTasks.ts

```diff
@@ -0,0 +1,80 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { existsSync } from "fs";
+import { LeafWithDoneFileTask } from "./leafTask";
+import path from "path";
+import { readFileAsync } from "../../../common/utils";
+import { GitRepo } from "../../../common/gitRepo";
+
+export class FlubListTask extends LeafWithDoneFileTask {
+	private getResourceGroup() {
+		const split = this.command.split(" ");
+		const index = split.indexOf("-g");
```

Comment:
```suggestion
		const index = split.indexOf("-g");
```
nit: `--releaseGroup` is also valid.

Reply:
Changed.